### ISSUE TITLE
Option to record source Tweets

### ISSUE THREAD

**Issue by Krunklehorn** [RipMeApp/ripme#1392]
* Ripme version: v1.7.86 
* Java version: java version "1.8.0_221"
Java(TM) SE Runtime Environment (build 1.8.0_221-b11)
Java HotSpot(TM) Client VM (build 25.221-b11, mixed mode, sharing)
* Operating system: Microsoft Windows [Version 10.0.18362.30]

* Exact URL you were trying to rip when the problem occurred: -Any Twitter user-

* Please include any additional information about how to reproduce the problem:

Just switched over to Ripme after the changes to Twitter's layout rendered the Twitter Media Downloader extension for Chrome obsolete. That extension worked well while it lasted, since it kept track of direct links as well as the original tweets they came from.

Here is an example of what Twitter Media Downloader's output looked like...

1. 2019/06/19 18:00:06 https://twitter.com/ghangaji_/status/1141465707227447296
  img1) https://pbs.twimg.com/media/D9dNAjtU4AAwco_?format=jpg&name=orig
2. 2019/06/18 20:48:24 https://twitter.com/ghangaji_/status/1141145673871441920
  img1) https://pbs.twimg.com/media/D9Yp8EGVUAARmQF?format=jpg&name=orig
3. etc. ....

As shown, direct links are parented to the source Tweet they came from, keeping things nice and organized. It'd be great if Ripme could do the same.

## Desired Behavior

For Ripme to include links to source Tweets as an option or part of the default behavior.

## Actual Behavior

Ripme only records direct links, not source Tweets.

**Comment by Isaaku** [comment]
Do you want to show in the log to show the parent tweet with the images download from there?

**Comment by Krunklehorn** [comment]
Yes, either in the log or in the URLs.txt, or both. That way we can find the source tweet if we have an image URL, or vice versa.
